ghost/security/lib/string.js

```diff
@@ -1,10 +1,8 @@
-const _ = require('lodash');
 const slugify = require('@tryghost/string').slugify;
 
-module.exports.safe = function safe(string, options) {
-    options = options || {};
+module.exports.safe = function safe(string, options = {}) {
     let opts = {requiredChangesOnly: true};
-    if (!_.has(options, 'importing') || !options.importing) {
+    if (!('importing' in options) || !options.importing) {
         opts.requiredChangesOnly = false;
     }
```

Comment:
From https://github.com/TryGhost/Utils/pull/147/files#r613351205
```js
    const opts = {requiredChangesOnly: options.importing === true};
```

<html>
<body>
<!--StartFragment--><p dir="auto" style="tab-size: var(--tab-size)  !important; box-sizing: border-box; margin-top: 0px; margin-bottom: 16px; color: rgb(201, 209, 217); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(13, 17, 23); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">I think this is the same exact logic (since<span> </span><code class="notranslate rgh-observing-whitespace rgh-linkified-code" style="tab-size: var(--tab-size)  !important; box-sizing: border-box; font-family: ui-monospace, SFMono-Regular, &quot;SF Mono&quot;, Menlo, Consolas, &quot;Liberation Mono&quot;, monospace; font-size: 11.9px; padding: 0.2em 0.4em; margin: 0px; background-color: var(--color-neutral-muted); border-radius: 6px;">undefined<span data-rgh-whitespace="space" style="tab-size: var(--tab-size)  !important; box-sizing: border-box; line-height: 1em; background-clip: border-box; background-repeat: repeat-x; background-position: left center; background-image: url(&quot;data:image/svg+xml,%3Csvg preserveAspectRatio=\&quot;xMinYMid meet\&quot; viewBox=\&quot;0 0 12 24\&quot; xmlns=\&quot;http://www.w3.org/2000/svg\&quot;%3E%3Cpath d=\&quot;M4.5 11C4.5 10.1716 5.17157 9.5 6 9.5C6.82843 9.5 7.5 10.1716 7.5 11C7.5 11.8284 6.82843 12.5 6 12.5C5.17157 12.5 4.5 11.8284 4.5 11Z\&quot; fill=\&quot;rgba%28128, 128, 128, 50%25%29\&quot;/%3E%3C/svg%3E&quot;); background-size: 1ch 1.25em;"><span> </span></span>!==<span data-rgh-whitespace="space" style="tab-size: var(--tab-size)  !important; box-sizing: border-box; line-height: 1em; background-clip: border-box; background-repeat: repeat-x; background-position: left center; background-image: url(&quot;data:image/svg+xml,%3Csvg preserveAspectRatio=\&quot;xMinYMid meet\&quot; viewBox=\&quot;0 0 12 24\&quot; xmlns=\&quot;http://www.w3.org/2000/svg\&quot;%3E%3Cpath d=\&quot;M4.5 11C4.5 10.1716 5.17157 9.5 6 9.5C6.82843 9.5 7.5 10.1716 7.5 11C7.5 11.8284 6.82843 12.5 6 12.5C5.17157 12.5 4.5 11.8284 4.5 11Z\&quot; fill=\&quot;rgba%28128, 128, 128, 50%25%29\&quot;/%3E%3C/svg%3E&quot;); background-size: 1ch 1.25em;"><span> </span></span>true</code>), but it's way more readable (not going to lie, took me longer than I would like to admit to understand what the original impl. was doing<span> </span><g-emoji class="g-emoji" alias="grimacing" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/1f62c.png" style="tab-size: var(--tab-size)  !important; box-sizing: border-box; display: inline-block; min-width: 1ch; font-family: &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;, &quot;Segoe UI Symbol&quot;; font-size: 1em; font-weight: 400; line-height: 1; vertical-align: -0.075em; font-style: normal !important;">😬</g-emoji>)</p>

importing in options | options.importing | requiredChangesOnly
-- | -- | --
false | X (doesn't matter) | false
true | false | false
true | true | true

<!--EndFragment-->
</body>
</html>

Reply:
👍 LGTM